Marlin/src/pins/stm32f1/pins_FLY_MINI.h

```diff
@@ -0,0 +1,190 @@
+
+
+
+
+//
+// Steppers
+//
+#define X_STEP_PIN                          PB1
+#define X_DIR_PIN                           PB2
+#define X_ENABLE_PIN                        PB10
+#ifndef X_CS_PIN
+  #define X_CS_PIN                          PB0
+#endif
+
+
+
+
+#define Y_STEP_PIN                          PA2
+#define Y_DIR_PIN                           PC4
+#define Y_ENABLE_PIN                        PC5
+#ifndef Y_CS_PIN
+  #define Y_CS_PIN                          PA7
+#endif
+
+
+#define Z_STEP_PIN                          PA3
+#define Z_DIR_PIN                           PA5
+#define Z_ENABLE_PIN                        PA6
+#ifndef Z_CS_PIN
+  #define Z_CS_PIN                          PA4
+#endif
+
+
+
+
+#define E0_STEP_PIN                         PA1
+#define E0_DIR_PIN                          PC3
+#define E0_ENABLE_PIN                       PA0
+#ifndef E0_CS_PIN
+  #define E0_CS_PIN                         PC2
+#endif
+
+#if ENABLED(TMC_USE_SW_SPI)
+  #ifndef TMC_SW_MOSI
+    #define TMC_SW_MOSI                     PB15
+  #endif
+  #ifndef TMC_SW_MISO
+    #define TMC_SW_MISO                     PB14
+  #endif
+  #ifndef TMC_SW_SCK
+    #define TMC_SW_SCK                      PB13
+  #endif
+#endif
+
+#if HAS_TMC_UART
+  
+  //
+  // Software serial
+  //
+  #define X_SERIAL_TX_PIN                   PB0
+  #define X_SERIAL_RX_PIN                   PB0
+
+  #define Y_SERIAL_TX_PIN                   PA7
+  #define Y_SERIAL_RX_PIN                   PA7
+
+  #define Z_SERIAL_TX_PIN                   PA4
+  #define Z_SERIAL_RX_PIN                   PA4
+
+  #define E0_SERIAL_TX_PIN                  PC2
+  #define E0_SERIAL_RX_PIN                  PC2
+
+#endif
+
+
+//
+// Heaters / Fans
+//
+#define HEATER_0_PIN                        PC6
+#define HEATER_BED_PIN                      PC7
+#ifndef FAN_PIN
+  #define FAN_PIN                           PC8
+#endif
+#define FAN1_PIN                            PC9
+//
+// Temperature Sensors
+//
+#define TEMP_BED_PIN                        PC0   // Analog Input
+#define TEMP_0_PIN                          PC1   // Analog Input
+
+
+
+
+//
+// LCD Pins
+//
+
+//
+// LCD / Controller
+//
+#define ENABLE_SPI2
+#define SS_PIN                             PB12
+#define SCK_PIN                             PB13
+#define MISO_PIN                            PB14
+#define MOSI_PIN                            PB15
+#define SDSS                                SS_PIN 
+#define SD_DETECT_PIN                       PB11
+#define BEEPER_PIN                          PC14
+
+#define LCD_PINS_RS                         PB8
+#define LCD_PINS_ENABLE                     PB9
+#define LCD_PINS_D4                         PB7
+#define LCD_PINS_D5                         PB6
+#define LCD_PINS_D6                         PB5
+#define LCD_PINS_D7                         PB4
+
+#define BTN_EN1                             PD2
+#define BTN_EN2                             PB3
+#define BTN_ENC                             PC13
```

Comment:
I think these need to be restricted to the type of display they are compatible with.